src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -65,42 +81,60 @@ const TabbedLayout: FunctionComponent<TabbedLayoutProps> = ({
         }
     };
 
+    const toggleCollapse = () => setCollapsed(!collapsed);
+
     return (
         <Box className={ classes.root } display="flex" height="100vh">
             <OrganizeSidebar />
             <Box display="flex" flexDirection="column" height="100vh" overflow="auto" position="relative" width={ 1 }>
                 <Box display={ fixedHeight ? 'flex' :'block' } flexDirection="column" height={ fixedHeight ? 1 : 'auto' }>
                     <Box component="header" flexGrow={ 0 } flexShrink={ 0 }>
-                        <Box mb={ 2 } pt={ 3 } px={ 3 }>
-                            <BreadcrumbTrail/>
-                            { /* Title, subtitle, and action buttons */ }
-                            <Box className={ classes.titleGrid } mt={ 2 }>
-                                <Box overflow="hidden">
-                                    <Typography className={ classes.title } component="h1" data-testid="page-title" noWrap variant="h3">
-                                        { title }
-                                    </Typography>
-                                    <Typography component="h2" variant="h5">
-                                        { subtitle }
-                                    </Typography>
-                                </Box>
-                                <Box>
-                                    { actionButtons }
+                        <Box mb={ collapsed ? 1 : 2 } pt={ 3 } px={ 3 }>
+                            <Box display="flex" flexDirection="row" justifyContent="space-between">
+                                <BreadcrumbTrail highlight={ collapsed } />
+                                <Box className="collapse-button">
+                                    <Button fullWidth onClick={ toggleCollapse } size="small"
+                                        startIcon={  <ArrowUpward /> }>
+                                        { intl.formatMessage({
+                                            id: 'layout.organize.header.collapseButton.' + (collapsed ? 'expand' : 'collapse'),
+                                        }) + ' ' + intl.formatMessage({
+                                            id: 'layout.organize.header.name',
+                                        }) }
```

Comment:
I think this is a case for a bit more complex use of the format strings in our internationalisation library.

First off I think you might want to use the `<FormattedMessage />` component. 

Then the text string can handle conditions IN the string, by accessing an option passed to the component

```
<FormattedMessage
  id="layout.organize.header.collapseButton"
  options={{
    collapsed
  }}
/>
```

Locale string:
```
collapseButton:  {collapsed, select, true {expand} false {collapse}} header
```



This is using the "ICU Message Format", I found an example of the switch select on this page: https://lokalise.com/blog/complete-guide-to-icu-message-format/#Switch_with_select. The contrived example is a stupid gender / pronouns one :D At least it has "they"!

Reply:
This may not be a perfect example but this is kinda the direction to go.

Reply:
Yeah, concatenating strings like that will not work in some languages. I imagine German probably has the verb (collapse/expand) after the noun (header), for example.

Either do what @djbusstop suggests and use ICU `select`, or create two distinct messages that contain the entire string, and switch between the two in TS (i.e what @lowlandjuju already did, except the concatenation). This might actually be easier, and result both in less code and message text that is easier for translators to translate:

```yml
header:
    collapseButton:
        collapse: collapse header
        expand: expand header
```

Reply:
Prefer the latter solution - repeating the word header is less verbose overall